M/Mineos/build_tarballs.jl

```diff
@@ -3,19 +3,37 @@
 using BinaryBuilder
 
 name = "Mineos"
-version = v"1.0"
+version = v"1.0.2" 
 
 # Collection of sources required to build Mineos
 sources = [
-    "https://github.com/anowacki/mineos.git" =>
-    "e2558b486d7656ef112608a8776643da66dc87cf",
```

Comment:
@anowacki why did you use a fork here?  Would using the upstream repo be ok now?

Reply:
@giordano The upstream repo has not been maintained for some years, and [my open issue](https://github.com/geodynamics/mineos/issues/10) hasn't received any response.  I can submit a PR and see if that gets any more attention at some point, but it seems fruitful for now to carry on pointing to the fork while we wait to see if anything will be accepted upstream.

(I should point out that the effort by @jeremiahpslewis to update this is completely independent of my own work in getting into Yggdrasil and I wasn't aware of it until now.)